src/gui/abstractfileitemwidget.cpp

```diff
@@ -38,10 +38,13 @@ static const int boxVSpacing = 10;
 static const int shadowBlurRadius = 20;
 static const QSize iconSize = QSize(15, 15);
 
+static const QColor folderIconColorLight = QColor("#666666");
```

Comment:
Not possible to use QSS instead ?

Reply:
I did not find a solution with qss because of call to method `KDC::GuiUtility::getIconWithColor`:
`QColor color = GuiUtility::isDarkTheme() ? folderIconColorDark : folderIconColorLight;
    _driveIconLabel->setPixmap(
        KDC::GuiUtility::getIconWithColor(":/client/resources/icons/actions/icon-folder-empty.svg", color).pixmap(iconSize));`

Reply:
What is done elsewhere is to use a Q_PROPERTY.
See AddDriveSmartSyncWidget:
Q_PROPERTY(QColor logo_color READ logoColor WRITE setLogoColor)

Reply:
Done